taxcalc/calcfunctions.py

```diff
@@ -547,86 +1033,188 @@ def ItemDed(e17500_capped, e18400_capped, e18500_capped, e19200_capped,
-    Taxpayer Characteristics:
-        e17500_capped : Medical expenses, capped by ItemDedCap
+    ID_Medical_frt: float
+        Floor (as decimal fraction of AGI) for deductible medical expenses
+
+    ID_Medical_frt_add4aged: float
+        Add on floor (as decimal fraction of AGI) for deductible medical expenses for elderly filing units
+
+    ID_Medical_hc: float
+        Medical expense deduction haircut
+
+    ID_Casualty_frt: float
+        Floor (as decimal fraction of AGI) for deductible causality loss
+
+    ID_Casualty_hc: float
+        Casualty expense deduction haircut
+
+    ID_Miscellaneous_frt: float
+        Floor (as decimal fraction of AGI) for deductible miscellaneous expenses
+
+    ID_Miscellaneous_hc: float
+        Miscellaneous expense deduction haircut
+
+    ID_Charity_crt_all: float
+        Ceiling (as decimal fraction of AGI) for all charitable contribution deductions
+
+    ID_Charity_crt_noncash: Ceiling (as decimal fraction of AGI) for noncash charitable contribution deductions
+
+    ID_prt: float
+        Itemized deduction phaseout rate
+
+    ID_crt: float
+        Itemized deduction maximum phaseout as a decimal fraction of total itemized deductions
+
+    ID_c: float
+        Ceiling on the amount of itemized deductions allowed (dollars)
+
+    ID_StateLocalTax_hc: float
+        State and local income and sales taxes deduction haircut
+
+    ID_Charity_frt: float
+        Floor (as decimal fraction of AGI) for deductible charitable contributions
+
+    ID_Charity_hc: float
+        Charity expense deduction haircut
+
+    ID_InterestPaid_hc: float
+        Interest paid deduction haircut
 
-        e18400_capped : State and local taxes, capped by ItemDedCap
+    ID_RealEstate_hc: float
+        State, local, and foreign real estate taxes deductions haircut
 
-        e18500_capped : Real-estate taxes, capped by ItemDedCap
+    ID_Medical_c: float
+        Ceiling on the amount of medical expense deduction allowed (dollars)
 
-        e19200_capped : Interest paid, capped by ItemDedCap
+    ID_StateLocalTax_c: float
+        Ceiling on the amount of state and local income and sales taxes deduction allowed (dollars)
 
-        e19800_capped : Charity cash contributions, capped by ItemDedCap
+    ID_RealEstate_c: float
+        Ceiling on the amount of state, local, and foreign real estate taxes deduction allowed (dollars)
 
-        e20100_capped : Charity noncash contributions, capped by ItemDedCap
+    ID_InterestPaid_c: float
+        Ceiling on the amount of interest paid deduction allowed (dollars)
 
-        e20400_capped : Total miscellaneous expenses, capped by ItemDedCap
+    ID_Charity_c: float
+        Ceiling on the amount of charity expense deduction allowed (dollars)
 
-        g20500_capped : Gross casualty or theft loss (before disregard),
-                        capped by ItemDedCap
+    ID_Casualty_c: float
+        Ceiling on the amount of casualty expense deduction allowed (dollars)
+
```

Comment:
@amshoulders You don't need a space between parameters in these lists. To save lines, it'd be better to not have them, IMO.

Reply:
Good call, thanks.  I will remove the extra lines.